modules/openapi-generator/src/main/resources/csharp-netcore/libraries/generichost/JsonConverter.mustache

```diff
@@ -323,4 +329,4 @@
 
             writer.WriteEndObject();
         }
-    }
```

Comment:
Please remove this spacing change. Ill look at the other generichost changes later. Thanks for the work here.

Reply:
I want to. Never did I want to add this extra line but looks like that's a limitation in mustache.

Let me try other combination of mustache tags to see if I've any luck

Reply:
Should be fixed via https://github.com/OpenAPITools/openapi-generator/pull/15377/commits/d8a11445a0f10abb6524de7dfe2f660d68947c48